### PR TITLE
Add link to Person page (SAI-13)

### PR DESCRIPTION
**Pull request by gniezen**
Adds a link to the Person page as described in [SAI-13].

### CODE REVIEW

File: assets/main.js [hunk 1]
```diff
@@ -38,7 +40,7 @@ function requestRollbarInfo(client, id) {
   const settings = {
     url: 'https://api.rollbar.com/api/1/rql/jobs/',
     data: {
-     query_string: 'SELECT item.title, timestamp, occurrence_id, project_slug, item.counter from item_occurrence WHERE timestamp > unix_timestamp() - 60 * 60 * 24 * 30 AND person.email="' + id + '" LIMIT 10',
+     query_string: 'SELECT item.title, timestamp, occurrence_id, project_slug, item.counter, person.id, person.username from item_occurrence WHERE timestamp > unix_timestamp() - 60 * 60 * 24 * 30 AND person.email="' + id + '" LIMIT 10',
```
Comment (pazaan) [inline assets/main.js]: Prefer templated string.

File: assets/main.js [hunk 2]
```diff
@@ -109,6 +111,9 @@ function getResults(client, response, data) {
     data.rows = [];
 
     client.metadata().then(function(metadata) {
+      data.personUrl = 'https://rollbar.com/' + metadata.settings.account_name + '/' + rows[0][3] + '/person/?environment=production&person_id=' + rows[0][5];
```
Comment (pazaan) [inline assets/main.js]: Prefer templated string.

**Review by pazaan — APPROVED** [review]
Couple of suggestions, but LGTM.

**Review by pazaan — APPROVED** [review]
LGTM! :shipit:

**Comment by gniezen** [comment]
Private app on Zendesk has been updated with the latest code.

**Pull request merged**